lib/mergometer/report.rb

```diff
@@ -46,13 +46,19 @@ def entries
       end
 
       def filter
-        "base:master repo:#{repo} type:pr is:merged"
+        "base:master #{repo_query} type:pr is:merged"
       end
 
       def fields_to_preload
         []
       end
 
+      def repo_query
+        @_query = repo.split(',').map do |r|
```

Comment:
Should we use the same naming for the method / memoised variable?

(Nit: double quotes 😉)

Reply:
Oops